indico/htdocs/js/indico/modules/registration/form/tpls/toolbar.tpl.html

```diff
@@ -25,3 +34,9 @@
             class="i-button icon-stack-minus" title="{{ 'Collapse All' | i18n }}"></a>
     </div>
 </div>
+
+<script>
+    $(document).ready(function() {
+        $('.js-dropdown').closest('.group').dropdown();
```

Comment:
And I'd use a more specific selector unless the dropdown js is fine with being called again after having been initialized